### ISSUE TITLE
Localise the decimal separator

### ISSUE THREAD

**Issue by BasO** [iina/iina#2597]
**System and IINA version:**

- macOS 10.15 beta 10
- IINA 1.0.4 build 107

**Expected behavior:**

The decimal signifier is either a decimal or a dot, depending on where you live. If you use a localised version of IINA, you'd expect IINA to apply the correct sign in dynamically set values (`%.1`, `%.2`) e.g. the audio delay in the Menu and OSC. For Dutch, you'd expect to find a comma, e.g. `Audiovertraging: 0,00s` 

**Actual behavior:**

These numbers currently show up with the incorrect (American) dot separator for strings with dynamic numbers.

- [ ] MPV does not have this problem (not sure).

**How often does this happen?**
Always for Dutch. Other localisations of IINA that may likewise be affected are:

- Danish
- French
- German
- Italian
- Polish
- Russian
- Spanish
- Turkish
- Ukrainian
- Slovak
- Czech

**Comment by anohren** [comment]
And Swedish.

Issue already exists: https://github.com/iina/iina/issues/2090

**Comment by alejx** [comment]
Are there other conventions about numbers in your languages? If so, then I guess it might be necessary to use Number Formatter and set the locale to the current locale before we display any number on the user interface.

**Comment by anohren** [comment]
Number formatter is the way to go. I think I confused that with string with format in the existing issue.

**Comment by BasO** [comment]
I agree, and @anohren I imagine it's a common source of confusion when the function called localizedStringWIthFormat doesn't localize the format contents (: . The decimal signifier is what people are more sensitive to (even Spotlight calculator is strict about using the correct one for your locale) but the 'thousands operator' also differs per locale. [Examples](https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use).
